packages/server/src/api/controllers/deploy/index.ts

```diff
@@ -148,6 +163,88 @@ export async function deploymentProgress(
   }
 }
 
+export async function publishStatus(ctx: UserCtx<void, PublishStatusResponse>) {
+  if (!(await features.isEnabled(FeatureFlag.WORKSPACE_APPS))) {
+    return (ctx.body = { automations: {}, workspaceApps: {} })
+  }
+  try {
+    const prodDb = context.getProdAppDB()
```

Comment:
It would be better to have this living in the sdk, extracting the koa objects and being able to easily write tests for it